frontend/src/lib-components/atoms/SrOnly.tsx

```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import styled from 'styled-components'
+
+export const SrOnly = styled.div`
```

Comment:
I'd love for this to be named `ScreenReaderOnly` instead. As it is now, you need to know what Sr means and I was only able to figure this out by the context of this PR.